pkg/imager/download.go

```diff
@@ -22,7 +26,13 @@ func Download(log logrus.FieldLogger, url, dst string, total, current *int64) er
 	defer closeF()
 
 	// Prepare download response.
-	resp, err := http.Get(url)
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
+	if err != nil {
+		return err
+	}
+	client := &http.Client{}
```

Comment:
Can we make this one global? I know that global vars is not the best thing to do, and it doesn't change much, but properly one allocation less is worth it. But it's just a suggestion anyway

Reply:
@Darkren `http.DefaultClient` could be used instead, which is global within the `http` package.

Reply:
Thanks for the review!

It would probably be better to postpone any optimizations until we actually see that an operation has issues with performance.

I'm not sure it will make any save performance-wise, though. Since the client is local to the function it will be allocated/deallocated on stack automatically, so no allocation costs in terms of speed should happen. Unless I miss something?